src/main/java/org/ecocean/Shepherd.java

```diff
@@ -4853,5 +4853,25 @@ public StoredQuery getStoredQuery(String uuid) {
     return listy;
   }
 
+  public List<Decision> getDecisionsForEncounter(Encounter enc){
+    try {
+      String queryString = "SELECT FROM org.ecocean.Decision WHERE this.encounter.catalogNumber == '"+enc.getCatalogNumber()+"'";
+      Query query = pm.newQuery(queryString);
+      query.setClass(Decision.class);
+      List<Decision> decisions = (List<Decision>) query.execute();
+      if (decisions!=null && decisions.size()>0) return decisions;
```

Comment:
not sure how "bad" it is to return here _before_ you do the `query.closeAll()` ... but i think these should be flipped around here?